src/lib/Components/Bidding/Screens/BidResult.tsx

```diff
@@ -9,16 +10,37 @@ import { BiddingThemeProvider } from "../Components/BiddingThemeProvider"
 import { Button } from "../Components/Button"
 import { CenteringContainer, Container } from "../Components/Containers"
 import { Divider } from "../Components/Divider"
+import { MarkdownRenderer } from "../Components/MarkdownRenderer"
 import { Timer } from "../Components/Timer"
 
+// import { SelectMaxBid_sale_artwork } from "__generated__/SelectMaxBid_sale_artwork.graphql"
+import { BidResult_sale_artwork } from "__generated__/BidResult_sale_artwork.graphql"
+
+// interface MinimumNextBid {
+//   display: string
+//   cents: number
+// }
+
 interface BidResultProps {
+  sale_artwork: BidResult_sale_artwork
   winning: boolean
   message_header?: string
   message_description_md?: string
+  navigator: NavigatorIOS
 }
 
 export class BidResult extends React.Component<BidResultProps> {
+  onPressBidAgain = () => {
+    // pushing to MaxBidScreen creates a sircular relay reference but this works
```

Comment:
/cc @alloy I don't think this is a problem, but we did run into this. Just curious if you'd seen any other cases where Relay circular references were an issue?

Reply:
Yes exactly. We essentially have this user flow:

```
SelectMaxBid -> ConfirmBid -> BidResult -> SelectMaxBid
```

And the fragments for each of these have reference the next, which forms a cycle. It's the fragments' cycle that the Relay compiler doesn't like, but removing the `...SelectMaxBid_sale_artwork` from the `BidResult_sale_artwork` fragment causes missing fragment data to be included in any push to `SelectMaxBid`. I'm thinking that popping _back_ to `SelectMaxBid` makes total sense, I'm just flagging this as an interesting limitation.

Reply:
There has been a commit that showed some circular reference (well, actually self referential) but I’m not sure it related to this case exactly https://github.com/facebook/relay/commit/79854495cc7eb47feb026c43389f3b1a43dbc4ec#diff-f73d2080c02a4b4398843309ebf8bce5R367